Makefile

```diff
@@ -9,19 +9,15 @@ docs: ## Build HTML documentation
 	$(MAKE) -C docs
 
 compile-requirements: ## Compile requirements files
-	pip-compile requirements/base.in
-	pip-compile requirements/dev.in
-	pip-compile requirements/docs.in
+	pip-compile ${COMPILE_OPTS} requirements/base.in
+	pip-compile ${COMPILE_OPTS} requirements/dev.in
+	pip-compile ${COMPILE_OPTS} requirements/docs.in
```

Comment:
What's the reason for having COMPILE_OPTS? What values can it have?

Reply:
This is a technique that I also used in edx-platform to de-duplicate the list of `*.in` files in the Makefile.